aweber_api/aweber.php

```diff
@@ -122,6 +122,8 @@ protected function readResponse($response, $url) {
         $this->adapter->parseAsError($response);
         if (!empty($response['id'])) {
             return new AWeberEntry($response, $url, $this->adapter);
+	} else if (!empty($response['broadcast_id'])) {
```

Comment:
Going a step further, it would be more DRY to have

```
if (!empty($response['id']) || !empty($response['broadcast_id'])) {
...
```


and only have one return statement for AWeberEntry.